sbpy/spectroscopy/tests/test_spec_remote.py

```diff
@@ -0,0 +1,50 @@
+import astropy.units as u
+from ..spectroscopy import nophotod_prodrate
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+import numpy as np
+
+
+@remote_data
+def test_remote_prodrate_simple():
+
+    hcn = Table.read('data/HCN.csv', format="ascii.csv")
+
+    temp_estimate = 33. * u.K
+
+    object = '900918'
```

Comment:
I found that they occasionally update the Horizons target identifiers so it may be better to use the comet name that jplhorizons understands

Reply:
When using the comet name Horizons complains because there's more than one apparition and asks us to use a more unique identifier. That was the first error we were getting with Ephem too, it was just hard to understand from the error message given.

Reply:
Horizons and `astroquery.jplhorizons` have a current apparition flag that should retrieve the most recent orbit so that this error can be avoided.

Reply:
I'll check that out, thanks @mkelley